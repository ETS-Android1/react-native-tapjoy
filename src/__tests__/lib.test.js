import * as React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme';
import toJson, { createSerializer } from 'enzyme-to-json';
import { ExampleComponent } from '../../example/ExampleComponent';

expect.addSnapshotSerializer(createSerializer({ mode: 'deep' }));

configure({ adapter: new Adapter() });

describe('React Native Tapjoy', () => {
  test('renders with basic hooks', () => {
    const wrapper = shallow(<ExampleComponent />);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
