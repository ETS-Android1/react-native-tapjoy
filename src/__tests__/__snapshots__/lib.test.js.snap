// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`React Native Tapjoy renders with basic hooks 1`] = `<View />`;
